sympy/integrals/tests/test_laplace.py

```diff
@@ -488,6 +492,16 @@ def test_laplace_transform():
     assert LT(Mt, t, s, legacy_matrix=False, noconds=True) == Ms
 
 
+@slow
+@XFAIL
+def test_laplace_transform_exp_2t2():
+    LT = laplace_transform
+    b = symbols('b', positive=True)
+    t = symbols('t')
+    assert (LT(b*exp(2*t**2), t, s) ==
+            (b*LaplaceTransform(exp(2*t**2), t, s), -oo, True))
```

Comment:
There should be a comment here that the test belongs in `test_laplace_transform` once fixed and is already there but commented out.

For an XFAIL test I think it is better to assert that the output is not whatever it returns like:
```
In [9]: wrong = (b*(oo*sign(exp(-s**2/8)) + sqrt(2)*sqrt(pi)*exp(-s**2/8)*erfi(sqrt(2)*s/4)/4), -oo, True)

In [10]: assert LT(b*exp(2*t**2), t, s) != wrong
---------------------------------------------------------------------------
AssertionError
```
Then if the output ever changes to something else we will know.